pmda/simple.py

```diff
@@ -0,0 +1,126 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+"""
+Parallel Analysis helper --- :mod:`pmda.simple`
+===============================================
+
+Collection of decorators to analyze
+
+"""
+from __future__ import absolute_import
+
+from MDAnalysis.core.groups import AtomGroup
+from MDAnalysis.core.universe import Universe
+from MDAnalysis.coordinates.base import ProtoReader
+import numpy as np
+
+from .parallel import ParallelAnalysisBase
+
+
+class AnalysisFromFunction(ParallelAnalysisBase):
+    """
+    Create an analysis from a function working on AtomGroups
+
+    Attributes
+    ----------
+    results : ndarray
+        results of calculation are stored after call to ``run``
+
```

Comment:
Make clear what the call signature of `function` has to be. This is pretty crucial because the whole approach relies on `function` following certain guide lines. Spell out the guide lines.